### PR TITLE
Now serialize effect layers in scene serializer + parse in babylon file loader

### PR DESCRIPTION
**Pull request by julien-moreau**
TODO: update documentation (.babylon file specification)
Do you think we have to deal with AssetContainer for effect layers ?

### CODE REVIEW

**Comment by julien-moreau** [comment]
@deltakosh ok
Going to define .addEffectLayer and .removeEffectLayer in scene

**Comment by julien-moreau** [comment]
Done,
Tested in the Babylon.js Editor (serialize + parse) for highlight layer and glow layer

File: dist/preview release/what's new.md [hunk 1]
```diff
@@ -112,6 +112,7 @@
 - Add support for normalized and non-float data to `Buffer` and `VertexBuffer`. ([bghgary](https://github.com/bghgary)]
 - Added unlit material extension support to glTF 2.0 loader. ([bghgary](https://github.com/bghgary))
 - (Viewer) Viewer's declaration file automatically generated ([RaananW](https://github.com/RaananW))
+- Now serialize and parse effect layers of scene (HighLight and Glow layers) ([julien-moreau](https://github.com/julien-moreau))
```
Comment (deltakosh) [inline dist/preview release/what's new.md]: New.....functions for effect layers

File: src/Layer/babylon.highlightLayer.ts [hunk 2]
```diff
@@ -668,5 +673,86 @@
...
+                        serializationObject.excludedMeshes.push(excludedMesh.mesh.id);
+                    }
+                }
+            }
+
+            return serializationObject;
+        }
+
+        /**
+         * Creates a HighLight layer from parsed HighLight layer data
```
Comment (deltakosh) [inline src/Layer/babylon.highlightLayer.ts]: Highlight and now HighLight:)

**Comment by deltakosh** [comment]
Wonderful!

**Comment by sevan** [comment]
Please stop pinging me for review, this is not the sevan you are looking for.

**Comment by deltakosh** [comment]
Sorry about that!!
Pinging @sebavan

**Comment by sebavan** [comment]
Look good to me :-)

**Pull request merged**